en-US/firefox/whatsnew_70.lang

```diff
@@ -0,0 +1,72 @@
+## NOTE: https://bedrock-demo-achurchwell.oregon-b.moz.works/firefox/70.0/whatsnew
+
```

Comment:
Add another NOTE with the URL including variable %LOCALE%

Reply:
We need both links, like these two on the top of the page: https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/welcome/page1.lang. They serve different purposes.